packages/fxa-settings/src/components/FormPasswordWithBalloons/index.test.tsx

```diff
@@ -40,7 +40,10 @@ describe('FormPasswordWithBalloons component', () => {
     renderWithLocalizationProvider(<Subject passwordFormType="reset" />);
     const newPasswordField = screen.getByLabelText('New password');
 
-    fireEvent.focus(newPasswordField);
+    act(() => {
+      fireEvent.focus(newPasswordField);
```

Comment:
Nothing to do in your PR but just to share, @dschom and I realized this morning [the docs](https://testing-library.com/docs/dom-testing-library/api-events/) now say:

> Most projects have a few use cases for fireEvent, but the majority of the time you should probably use [@testing-library/user-event](https://testing-library.com/docs/user-event/intro).